lib/src/api/opt/capabilities.rs

```diff
@@ -16,6 +23,31 @@
 ///
 /// The capabilities are defined using allow/deny lists for fine-grained control.
 ///
+/// # Filtering functions and net-targets.
+///
+/// The filtering of net targets and functions is done with an allow/deny list.
+/// These list can either match everything, nothing or a given list.
+///
+/// By default every function and net-target is disallowed. For a function or net target to be
+/// allowed it must match the allow-list and not match the deny-list. This means that if for
+/// example a function is both in the allow-list and in the deny-list it will be disallowed.
+///
+/// With the combination of both these lists you can filter subgroups. For example:
+/// ```
+/// # use surrealdb::opt::capabilities::CapabilitiesBuilder;
```

Comment:
@DelSkayn I think you forgot to update the docs. They are still using `CapabilitiesBuilder`.